tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -1635,69 +1618,692 @@ TEST_F( HVACFixture, VRFTest_SysCurve_WaterCooled ) {
 			"  1.5,                     !- Maximum Curve Output",
 			"  Temperature,             !- Input Unit Type for X",
 			"  Dimensionless;           !- Output Unit Type",
+		} );
+
+		ASSERT_FALSE( process_idf( idf_objects ) );
+
+		DataGlobals::BeginEnvrnFlag = true;
+		DataSizing::CurZoneEqNum = 1;
+		DataEnvironment::OutBaroPress = 101325; // sea level
+		DataZoneEquipment::ZoneEquipInputsFilled = true; // denotes zone equipment has been read in
+		StdRhoAir = PsyRhoAirFnPbTdbW( DataEnvironment::OutBaroPress, 20.0, 0.0 );
+		ZoneEqSizing.allocate( 1 );
+		ZoneSizingRunDone = true;
+		ZoneEqSizing( CurZoneEqNum ).DesignSizeFromParent = false;
+		ZoneEqSizing( CurZoneEqNum ).SizingMethod.allocate( 25 );
+		ZoneEqSizing( CurZoneEqNum ).SizingMethod( DataHVACGlobals::SystemAirflowSizing ) = DataSizing::SupplyAirFlowRate;
+
+		FinalZoneSizing.allocate( 1 );
+		FinalZoneSizing( CurZoneEqNum ).DesCoolVolFlow = 0.566337; // 400 cfm * 3 tons = 1200 cfm
+		FinalZoneSizing( CurZoneEqNum ).DesHeatVolFlow = 0.566337;
+
+		ZoneSysEnergyDemand.allocate( 1 );
+
+		ProcessScheduleInput(); // read schedules
+		GetCurveInput(); // read curves
+		GetZoneData( ErrorsFound ); // read zone data
+		EXPECT_FALSE( ErrorsFound ); 
+
+		GetZoneEquipmentData(); // read equipment list and connections
+		GetVRFInputData( ErrorsFound );
+		EXPECT_TRUE( ErrorsFound ); 
+		EXPECT_EQ( 0, VRFTU( VRFTUNum ).VRFSysNum );
+		EXPECT_EQ( 0, VRFTU( VRFTUNum ).ZoneNum );
+		EXPECT_EQ( 0, VRFTU( VRFTUNum ).TUListIndex );
+		EXPECT_EQ( 0, VRFTU( VRFTUNum ).IndexToTUInTUList );
+
+		// clean up
+		ZoneSysEnergyDemand.deallocate();
+
+	}
+
+	TEST_F( HVACFixture, VRFTest_SysCurve_WaterCooled ) {
```

Comment:
@rraustad I think the unit tests are ok, just the diff is being stupid because they are so similar.  Check the full file in this branch, but I think the diff is showing the existing test being changed to the new test and then the old test being added back in here.